plugins/catalog/src/components/CatalogTable/CatalogTable.test.tsx

```diff
@@ -53,10 +55,24 @@ const entities: Entity[] = [
 ];
 
 describe('CatalogTable component', () => {
-  const mockApis = TestApiRegistry.from([
-    starredEntitiesApiRef,
-    new MockStarredEntitiesApi(),
-  ]);
+  const mockApis = TestApiRegistry.from(
+    [starredEntitiesApiRef, new MockStarredEntitiesApi()],
+    [
+      catalogApiRef,
+      {
+        getEntityFacets: jest.fn().mockResolvedValue({
```

Comment:
This is one of those places where [a `catalogApiMock`](https://github.com/backstage/backstage/blob/51b2573091a1f8807c8180c8a9c56eb9eb0779f4/plugins/catalog-react/src/testUtils/catalogApiMock.ts#L61) with a single entity in it would simplify things.